addons/azureSets/azuresets.lua

```diff
@@ -247,8 +268,11 @@ windower.register_event('addon command', function(...)
         elseif comm == 'help' then
             local helptext = [[AzureSets - Command List:')
             1. removeall - Unsets all spells.
-            2. spellset <setname> -- Set (setname)'s spells.
-            3. set <setname> -- Same as spellset
+            2. spellset <setname> (clearfirst|preservetraits) -- Set (setname)'s spells,
+                             optional parameter: clearfirst or preservetraits: overrides
+                             setting to clear spells first or remove individually,
+                             preserving traits where possible.
+            3. set <setname> (clearfirst|preservetraits) -- Same as spellset
```

Comment:
Minor nitpick here: [square brackets] are usually used to indicate optional arguments (&lt;angle brackets&gt; for required arguments). I would also prefer if the help text said exactly what the default was.